src/persist/event_store.rs

```diff
@@ -330,7 +330,7 @@ pub(crate) mod shared_test {
     };
     use crate::{Aggregate, DomainEvent};
 
-    #[derive(Debug, Serialize, Deserialize, PartialEq, Clone)]
+    #[derive(Debug, Serialize, Deserialize, PartialEq, Clone, Copy)]
```

Comment:
Copy works here because these are simple events meant only for demonstration, in most cases I think that `Copy` would be inappropriate here. Since this is primary here for others to learn, we should not add it to the derive.